.golangci.yml

```diff
@@ -1,19 +1,34 @@
-run:
-  timeout: 15m
+formatters:
+  enable:
+    - gci
+    - gofumpt
+
+  exclusions:
+    generated: strict
 
-output:
-  sort-results: true
+  settings:
+    gci:
+      sections:
+        - standard
+        - default
+        - prefix(github.com/prometheus/otlptranslator)
+    gofumpt:
+      extra-rules: true
+
+issues:
+  max-issues-per-linter: 0
+
+  max-same-issues: 0
 
 linters:
   # Keep this list sorted alphabetically
   enable:
     - depguard
+    - errcheck
```

Comment:
Why is this added, it's enabled by default, isn't it? In any case, I don't see the need for changing this as part of the migration:
```suggestion
```